app/views/pages/details.html.erb

```diff
@@ -125,7 +125,7 @@
         <li> Pull requests that are regarded by a project maintainer as a hindrance vs. helping.</li>
         <li> Something that's clearly an attempt to simply +1 your pull request count for October.</li>
       </ul>
-      <p>Last but not least, one pull request to fix a typo is fine, but 5 pull requests to remove a stray whitespace is
+      <p>Last but not least, one pull request to fix a typo is fine, but 4 pull requests to remove a stray whitespace are
```

Comment:
```suggestion
      <p>Last but not least, one pull request to fix a typo is fine, but 4 pull requests to remove a stray whitespace is
```
Afaik, `is not` is correct here, as its saying the singular notion of "4 spammy PRs to remove whitespace" is not okay, instead of saying each of the "4 spammy PRs to remove whitespace" are not okay,.

Reply:
Done. Thanks!